custom_components/continuously_casting_dashboards/dashboard_caster.py

```diff
@@ -320,12 +320,11 @@ async def start(self):
                                     continue
                             else:
                                 _LOGGER.info(f"HA Dashboard is NOT currently being cast on {device_name}. Skipping...")
-                                continue
```

Comment:
I haven't tested this but will removing this cause any issues for any skipped displays?

Reply:
Ah, I seem to have missed the innere for loop in the while loop and assumed the continue concerns the while loop.

Nonetheless, if the last device in the loop is skipped in this case (I only tested this with one device...), the service immediatly rechecks the device because the sleep in the end is skipped, as that is part of the for loop. This leads to a high CPU use.

Reply:
Okay thanks! I'll test this tonight with 5 devices and get it approved if there's no issues 👍

Reply:
Apologies for the delay, i've tested this now on the latest HA version and it's working great. Thanks!